installer/templates/new/widgets/text/text.scss

```diff
@@ -19,14 +19,20 @@ $moreinfo-color:    rgba(255, 255, 255, 0.7);
 
   .more-info {
     color: $moreinfo-color;
+    font-size: 24px;
   }
 
   .updated-at {
     color: rgba(255, 255, 255, 0.7);
```

Comment:
Since we're going to display the updated-at, let's also have it styled like the other widgets `color: rgba(0, 0, 0, 0.3);`